var/spack/repos/builtin/packages/py-tensorflow-datasets/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTensorflowDatasets(PythonPackage):
+    """tensorflow/datasets is a library of datasets ready to use with
+    TensorFlow."""
+
+    homepage = "https://github.com/tensorflow/datasets"
+    pypi = "tensorflow-datasets/tensorflow-datasets-4.4.0.tar.gz"
+
+    version(
+        "4.4.0",
+        sha256="3e95a61dec1fdb7b05dabc0dbed1b531e13d6c6fd362411423d0a775e5e9b960",
+    )
+
+    depends_on("python@3:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.6:", type=("build", "run"))
```

Reply:
I wonder how did I managed to type that wrong?